configuration/amd64/docker-compose.yml

```diff
@@ -30,23 +30,33 @@ services:
     tty: true
 
   v2xhub:
-    image: usdotfhwaops/v2xhub:develop
+    image: usdotfhwaops/v2xhubamd:${V2XHUB_VERSION:-develop}
     container_name: v2xhub
     network_mode: host
     restart: always
     depends_on: 
       - db
     environment:
       - MYSQL_PASSWORD=/run/secrets/mysql_password
-      - INFRASTRUCTURE_ID=rsu_<J2735 MAP MESSAGE INTERSECTION ID>
-      - INFRASTRUCTURE_NAME=<RSU_NAME>
+      - SIMULATION_MODE=${SIMULATION_MODE:-false}
```

Comment:
The default values alleviate a need for a sample env. With non provided it will use the default values. I could add documentation to instruct users on how to generate a .env file